app/helpers/application_helper/button/physical_storage_timeline.rb

```diff
@@ -0,0 +1,14 @@
+class ApplicationHelper::Button::PhysicalStorageTimeline < ApplicationHelper::Button::EmsTimeline
+  def calculate_properties
+    super
+    self[:hidden] = true unless visible?
+  end
+
+  def visible?
+    @record.supports?(:timeline)
+  end
+
+  def disabled?
+    @error_message.present?
```

Comment:
Will this ever be true?  The parent class sets `@error_message` in a conditional block then checks for `.present?` but this never sets it so how could it ever be true?

Reply:
This file was a reminder from the first commit, it is deleted.